src/main.cpp

```diff
@@ -181,6 +181,17 @@ bool vcMain_TakeScreenshot(vcState *pProgramState)
   if (currSize.x - pProgramState->settings.screenshot.resolution.x > 32 || currSize.y - pProgramState->settings.screenshot.resolution.y > 32)
     return true;
 
+  udFindDir* poutputPath(nullptr);
```

Comment:
This is fine, but we typically don't initialize this way.
`udFindDir* pOutputPath = nullptr;`

Reply:
Understood